packages/react-core/src/components/DataList/__tests__/DataListAction.test.tsx

```diff
@@ -58,18 +57,19 @@ test(`Renders with class ${styles.dataListAction} when isPlainButtonAction = tru
         aria-labelledby="check-action-item2 check-action-action2"
         id="check-action-action2"
         aria-label="Actions"
+        data-testid="actions"
       >
         test
       </DataListAction>
     );
 
     if (visMod === 'hidden') {
-      expect(screen.getByText('test')).toHaveClass(styles.modifiers[`${visMod}`]);
+      expect(screen.getByTestId('actions')).toHaveClass(styles.modifiers[`${visMod}`]);
     }
-    expect(screen.getByText('test')).toHaveClass(styles.modifiers[`${visMod}OnSm`]);
-    expect(screen.getByText('test')).toHaveClass(styles.modifiers[`${visMod}OnMd`]);
-    expect(screen.getByText('test')).toHaveClass(styles.modifiers[`${visMod}OnLg`]);
-    expect(screen.getByText('test')).toHaveClass(styles.modifiers[`${visMod}OnXl`]);
-    expect(screen.getByText('test')).toHaveClass(styles.modifiers[`${visMod}On_2xl`]);
+    expect(screen.getByTestId('actions')).toHaveClass(styles.modifiers[`${visMod}OnSm`]);
+    expect(screen.getByTestId('actions')).toHaveClass(styles.modifiers[`${visMod}OnMd`]);
+    expect(screen.getByTestId('actions')).toHaveClass(styles.modifiers[`${visMod}OnLg`]);
+    expect(screen.getByTestId('actions')).toHaveClass(styles.modifiers[`${visMod}OnXl`]);
+    expect(screen.getByTestId('actions')).toHaveClass(styles.modifiers[`${visMod}On_2xl`]);
   });
 });
```

Comment:
Can you add a test which asserts that the children of the DataListAction now have the dataListAction class?